### PR TITLE
Implement Renderable for Vec<T>

### PR DESCRIPTION
**Pull request by theduke**

### CODE REVIEW

File: render/src/simple_element.rs [hunk 1]
```diff
@@ -3,7 +3,7 @@ use crate::html_escaping::escape_html;
 use std::collections::HashMap;
 use std::fmt::{Result, Write};
 
-type Attributes<'a> = Option<HashMap<&'a str, &'a str>>;
+type Attributes<'a> = Option<HashMap<&'a str, String>>;
```
Comment (Schniz) [inline render/src/simple_element.rs]: I'm not sure why is this change needed — but I'm new to Rust. can you give me an example where `&'a str` isn't good enough?

Maybe for _returning_ a component other than rendering it?

File: render/src/render.rs [hunk 2]
```diff
@@ -50,6 +50,16 @@ impl<T: Render> Render for Option<T> {
     }
 }
 
+impl<T: Render> Render for Vec<T> 
+{
+    fn render_into<W: Write>(self, writer: &mut W) -> Result {
+        for elem in self {
+            elem.render_into(writer)?;
```
Comment (Schniz) [inline render/src/render.rs]: 👏

**Comment by vpzomtrrfrt** [comment]
@theduke what's the state of this?

File: render/src/simple_element.rs (hunk 1 above)
Comment (vpzomtrrfrt) [reply render/src/simple_element.rs:6]: I've encountered a few cases where it would be useful to pass owned values as attributes, but this change would cause unnecessary cloning. Maybe a Cow would be a good idea?

Comment (daaku) [reply render/src/simple_element.rs:6]: +1 undo this change since you're handling it in #21.

**Comment by daaku** [comment]
Just the impl for the `Vec<T>` seems good to merge imho.

**Comment by Schniz** [comment]
Merged in #40
